app/src/main/java/pl/droidcon/app/dagger/module/LogicModule.java

```diff
@@ -75,4 +77,29 @@ public ReminderPersistence provideReminderPersistence(Context context) {
     public Reminder provideReminder(Context context) {
         return new ReminderImpl(context);
     }
+
+    @Singleton
+    @Provides
+    public SingleEntityStore<Persistable> provideDatabase(Context context) {
+        DatabaseSource source = new DatabaseSource(context, Models.DEFAULT, 1);
```

Comment:
Ten sam kod `DatabaseSource source = new DatabaseSource(context, Models.DEFAULT, 1);` jest w `provideNonRxDatabase` - mozna to wyciagnac jako `@Provides` w Daggerze